uint8 tca_address
uint8 channel
uint8[] payload
---
bool success